src/Jobs/Middleware/SetQueuedExportLocale.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Maatwebsite\Excel\Jobs\Middleware;
+
+use Closure;
+use Illuminate\Contracts\Translation\HasLocalePreference;
+use Illuminate\Support\Traits\Localizable;
+
+class SetQueuedExportLocale
```

Comment:
Can you make this more generic, so it could also be used for imports?

Reply:
Maybe `SetJobLocal` and the variable `export` inside rename to `localizable` ?

Reply:
Middleware name: `LocalizeJob`
variable: `localizable`